node/test/graphql/mutation/login.spec.ts

```diff
@@ -98,6 +98,44 @@ describe('login', () => {
     );
   });
 
+  it(`returns user and updates token (using cookies to get auth token)`, async () => {
+    const date = new Date(Date.now() + 3000);
+    const tokenToStoreInCookie = getToken('5ffdf41a1ee2c62320b49ea1', 300);
+    const storedCookie = mockSetCookie(
+      'auth_token_cookie',
+      tokenToStoreInCookie,
+      7
+    );
+    const token = mockGetCookie(storedCookie, 'auth_token_cookie');
+    const response = await request(app)
+      .post('/graphql')
+      .send({
+        query: `mutation {
+          login(accessToken: "${token}") {
```

Comment:
we alway want to use the static form of gql queries, and then pass args through variables. 

We haven't gone back and fixed all the old tests in this way yet, but any new tests (or tests that get modified) need to use static queries

Reply:
Can you show me an example of a static form of gql queries?